src/lib/exeRunner.ts

```diff
@@ -4,6 +4,7 @@ import { spawn, spawnSync } from "child_process";
 import os = require('os');
 import getExePath from "./getExePath";
 import { Logger } from "./logger";
+import getAutomationAgent from "./runnerParameters"
```

Comment:
consider exporting just the 'getAutomationAgent' function inside "./runnerParameters" instead. That way it gets composed only when needed. Sure, technically it could be a globally set constant, but probably better to be computed at runtime iif needed.